.github/tpl/test.tpl.yml

```diff
@@ -83,6 +83,65 @@ jobs:
         pwd
 {{VM_RUN}}
 
+  testnfs:
+    if: ${{ !contains(github.repository, 'netbsd') && !contains(github.repository, 'openbsd') && !contains(github.repository, 'dragonflybsd') && !contains(github.repository, 'omnios')}}
```

Comment:
why exclude these vms,  can we support them?

Reply:
They should work, however I haven't looked into adding NFS to them in much detail since I do not currently use them in my GHA workflows, so this exclusion is to prevent tests from failing when not implemented. The NFS setup commands will be different from FreeBSD/Solaris.

If you would like NFS to be added to the other VMs before this PR is approved, it will take some time before I can look into it.

Reply:
we should include them, unless  nfs client is nsot avaliable on the OS.